[#1] 마이페이지 ui

gayeon00 opened the pull request:
## PR Type
마이페이지, 마이페이지 하위 ui입니당

<!-- 아래 괄호 사이"x"를 입력해서 체크할 수 있습니당. -->

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (프로젝트 설정)
- [ ] Documentation content changes
- [ ] Other... 


## PR Checklist
PR하기 전에 아래 조건 충족했는지 확인!

- [x] 커밋 메세지 포맷에 맞게 작성했나요?
- [ ] (bug fixes / features의 경우) docs을 첨부/수정 했나요?


## 변경사항
- 마이페이지 작성
- 산책 기록, 산책 상세 화면 작성
- recyclerview에 필요한 layout작성

## 스크린샷
![마이페이지](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/68911884/21176fe7-7275-41ac-bb09-5dcba1c11abe)

## Issue number and link
<!-- issue 링크 해주면 됩니다~-->
#1 

## 리뷰어들에게 전할 말
- 내 정보 수정, 반려견 등록, 반려견 정보 수정 부분은 회원가입 후 정보 입력이랑 겹쳐서 일단 남겨뒀습니다.
- 산책기록에 지도 부분도 진호님이 하신거 확인하고 하려고 남겨뒀습니다.
- RecyclerViewAdapter는 안만들었지만 recyclerView에 사용할 row_.xml은 다 작성해뒀습니다.
- 상세기록 날짜, 시작종료 시간 표시하는 ui가 고민이네요. 아이디어 있으면 환영입니다~~~

## 비고

gayeon00 committed:
feat: 마이페이지 ui

gayeon00 committed:
feat: 산책 기록 ui 마이페이지 수정

haehyun-lee reviewed (approved):
수고하셨습니다! 산책 기록 날짜같은건 원래 아래같이 위에 잘 보이게 넣는 게 좋다고 생각했는데, 
![KakaoTalk_20230907_084839863](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/86085387/89139197-3f21-4291-904e-c4e3f86ecefe)

생각해보니 저희 지도 크기도 화면 꽤 차지할 것 같아서 2번째 처럼 컴팩트하게 들어가는 것도 좋을 것 같습니다!
![KakaoTalk_20230907_084839863_01](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/86085387/d76986de-f6f4-4e8b-a7ae-b0b0b23fa2e4)

![KakaoTalk_20230907_084839863_02](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/86085387/4c6c77b5-817e-4ccc-8590-d4c442a33d2c)

gayeon00 committed:
feat: #1 산책 기록 ui 마이페이지 수정

- 마이페이지에 누적 산책 시간, 누적 산책거리, 내 평판(발바닥 점수) 보여주기
- 이달의 산책기록 항목 하나에 시작시간, 종료시간, 소요시간, 이동거리도 표시
- 산책기록 상세에 시작시간, 종료시간 표시, 날짜 크기 키움

gayeon00 commented:
피드백 주신거 반영해서 수정해봤습니다! 괜찮으시면 머지해주세용!

|산책 상세기록|이달의 산책기록 항목 하나|마이페이지|
|---|---|---|
|![Screenshot_1694054020](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/68911884/8ad7b1a0-d66c-44ee-aba8-9109b08f9d93)|![image](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/68911884/8987bedc-a9db-450b-a80e-8dcec35a2c87)|![Screenshot_1694054013](https://github.com/APPSCHOOL2-Android/FinalProject-PetPal/assets/68911884/a30c8212-5e4b-4078-ab41-63b72a64c559)|

gayeon00 committed:
Merge branch 'main' into gayeon/1-mypage-ui

wktkdandp commented:
고생하셨습니다!
확실히 ui가 이쁘네요
